docs/index.rst

```diff
@@ -0,0 +1,18 @@
+Scality S3 Server
+==================
+
+.. _user-docs:
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Documentation
+
+   GETTING_STARTED
+   CLIENTS
+   DOCKER
+   SSL
+   VERSIONING
+   HA
+   S3FS
+   DATA_METADATA_D
+   LISTING
```

Comment:
Since you deleted the data-metadata and listing documents, these two lines probably need to go as well, no?